### PR TITLE
Homepage hero banner

### LINKED ISSUE #77: Wireframe: Hero banner
Implement the top-level "Automation Powered by People" section from the wireframe issue #57 

This should result in a file such as `template/homepage-something-band.tmpl` along with the associated `themes/ansible-community/sass/_homepage-something-band.scss` definition.

Grab the bullito svg here: https://github.com/ansible/logos/blob/main/ansibull-community/ansibull-community.svg

Note that we can implement the nav/menu bar separately.

### PR DESCRIPTION
**Pull request by oraNod**
Resolves #77 

Styling needs work but we can do that in a follow up.

### CODE REVIEW

File: data/homepage.yaml [hunk 1]
```diff
@@ -12,7 +16,7 @@ bullhorn:
 platform:
   main: Get enterprise engineering and support from Red Hat with a platform that can tackle any automation challenge.
   discover: Ansible Automation Platform provides everything needed to create, execute, and manage automation in a single subscription. From execution environments to certified collections to automation analytics, discover the features and benefits of Ansible Automation Platform.
```
Comment (cybette) [inline data/homepage.yaml:18]: discover: Red Hat Ansible Automation Platform ...

(need the full name on first mention).

Comment (oraNod) [reply data/homepage.yaml:18]: This isn't the first mention on the page though. The key below "learn:" is used in the banner at the very top of the page. Likewise there is the full branded name in the logo.

I feel like, in the context of the rendered site, this is fine. I also see that the value of this key has two instances of "AAP" but still think Red Hat gets plenty of mentions here.

Comment (GregSutcliffe) [reply data/homepage.yaml:18]: The first use is in the hero band at the top, that is the full text. Is that close enough to this paragraph to count, @cybette? What's the official short-name we should use elsewhere?

File: themes/ansible-community/assets/images/unsplash.jpg
Comment (GregSutcliffe) [inline themes/ansible-community/assets/images/unsplash.jpg:1]: The top right corner of this is not our Pool colour, so while *most* of the image blends into the nav, the corner above Bullito doesn't. We should probably add a gradient or something - can wait for the styling pass, just wanted to note it.

**Review by GregSutcliffe — CHANGES_REQUESTED** [review]
Seems solid but we need to address the wording from @cybette

File: themes/ansible-community/templates/base.tmpl [hunk 2]
```diff
@@ -9,12 +9,16 @@
 {% endblock %}
 {{ template_hooks['extra_head']() }}
 </head>
-<body id="top">
+<body id="top" class="{% if permalink == '/' %}homepage{% endif %}">
```
Comment (GregSutcliffe) [inline themes/ansible-community/templates/base.tmpl:12]: this leaves and empty "class" parameter on other pages. Probably not harmful, but if we really want no class here, shouldn't the `class=` move inside the if?

Comment (oraNod) [reply themes/ansible-community/templates/base.tmpl:12]: oh, nice catch.

File: themes/ansible-community/assets/images/unsplash.jpg
Comment (oraNod) [reply themes/ansible-community/assets/images/unsplash.jpg:1]: Yeah this is a little tricky and I've spent a good while fiddling with the image. The original is blue and layering over to get it to blend nicely with the pool colour will probably take some more tweaking.

Here's the original https://unsplash.com/photos/xuTJZ7uD7PI

I had another go at it in inkscape and hopefully improved the result. We can go over it again.

**Review by cybette — APPROVED** [review]
LGTM, thanks @oraNod! (will raise a separate PR about the platform band)

**Review by GregSutcliffe — APPROVED** [review]

**Pull request merged**